runtime/development/src/lib.rs

```diff
@@ -1809,24 +1861,23 @@ pub type UncheckedExtrinsic =
 /// Extrinsic type that has already been checked.
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 
+parameter_types! {
+	// = 16.65 CFG per epoch (12h)
+	pub const CollatorRewards: Balance = 8_325 * MILLI_CFG;
+	// = 20,096 CFG per epoch (12h)
+	pub const TotalRewards: Balance = 10_048 * CFG;
+}
```

Comment:
These should be the correct numbers we eventually want to have on Centrifuge, taken from the [gov proposal](https://gov.centrifuge.io/t/rfc-add-a-block-reward-and-improve-collators-cycle/4766).